pkgs/applications/office/banana-accounting/default.nix

```diff
@@ -0,0 +1,56 @@
+{ autoPatchelfHook
+, cairo
+, config
+, e2fsprogs
+, fetchurl
+, gmp
+, gtk3
+, libGL
+, lib
+, pkgs
+, stdenv
+, xorg
+}:
+
+stdenv.mkDerivation {
+  pname = "banana-accounting";
+  version = "10.0.12";
+
+  srcs = fetchurl {
+    url = "https://www.banana.ch/accounting/files/bananaplus/exe/bananaplus.tgz";
+    hash = "sha256-Xs7K/Z6qM1fKKfYMkwAGznNR0Kt/gY7qTr8ZOriIdYw=";
+  };
+
+  sourceRoot = "./";
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  buildInputs = [
+    cairo
+    e2fsprogs
+    gmp
+    gtk3
+    stdenv.cc.cc.lib
+    libGL
+    xorg.libX11
+  ];
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -av . $out
+    ln -s $out/usr/bin/bananaplus $out/bin/
```

Comment:
Can't this be moved?

Reply:
Can this be moved instead of linked?

Reply:
If I try that, I get a popup with the following error:
```
"/nix/store/...-banana-accounting-10.0.12/Lang/"
Language directory not found.
Reinstall the program.
```
If you really think it's worth the effort, then I can try to look at other approaches to moving it, beyond the naive one, but given that Nix creates a bazillion links in everything it does, I don't quite see what the point would be of trying to avoid one more link.

In the meantime I've noticed that (regardless of whether `bin/bananaplus` is moved or linked to `$out/bin/`) the log says
```
2022-06-26 19:06:00.098	Start	BananaPlus 100012-220223; Linux NixOS 22.05 (Quokka) Linux; NixOS 22.05 (Quokka); en_GB; Qt5.15.8
2022-06-26 19:06:00.180	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.180	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.180	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.180	Warning	QNetworkReplyImplPrivate::error: Internal problem, this method must only be called once.
2022-06-26 19:06:00.180	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.180	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.180	Debug	"templateslist" "TLS initialization failed"
2022-06-26 19:06:00.185	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.185	Debug	"news" "TLS initialization failed"
2022-06-26 19:06:00.189	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.189	Debug	"highlights" "TLS initialization failed"
2022-06-26 19:06:00.194	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.194	Debug	"bananaapps" "TLS initialization failed"
2022-06-26 19:06:00.220	Warning	QSslSocket::connectToHostEncrypted: TLS initialization failed
2022-06-26 19:06:00.220	Debug	"news" "TLS initialization failed"
2022-06-26 19:06:02.122	Stop	
```
Any ideas what to do about this?

Reply:
Ok then.